File: Div2A/CF734-D2-A.cpp
#include <iostream>
#include <string.h>

int main() {
    int n, a(0),d(0);
    char x;

    std::cin>>n;

    while(n>0) {
        std::cin>>x;
        if (x == 'A') 
            a++;
        else
            d++;
        n--;
    }

    if (a>d)
        std::cout<<"ANTON";
    else if(a<d) 
        std::cout<<"DANIK";
    else 
        std::cout<<"Friendship";
}

File: Div2A/CF734-D2-A.py
string = [x for x in input()]
a = 0
d = 0

for x in string:
    if (x == 'A') :
        a += 1
    else:
        d += 1

if a>d:
    print("Anton")
elif a < d:
    print("Danik")
else:
    print("Friendship")
